### PR TITLE
Rephrased first few paragraphs of getting-started

### PR DESCRIPTION
**Pull request by mlohrer**
Hi there,

this is my first pull request. I thought I'd start with rephrasing some parts of the German translation of the React documentation. The idea is to optimise legibility, check for divergences with the current English-language original and stick to the previously established translations of React-specific concepts. All in all, a light-touch approach.

Thanks!

Martin

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,47 +18,47 @@ redirect_from:
...
+React wurde von Anfang an für eine schrittweise Einführung entwickelt und **du kannst selbst entscheiden, wie viel oder wie wenig du von React benutzen möchtest.** Die Links in diesem Abschnitt helfen dir beim Einstieg, ob du nur ein wenig Interaktivität zu einer kleinen HTML-Seite hinzufügen, bestehende Anwendungen erweitern oder eine komplexere Anwendung bauen möchtest.
 
-### Online Spielwiesen {#online-playgrounds}
+### Online-Sandkasten {#online-playgrounds}
 
-Wenn du ein bisschen mit React herum spielen möchtest, dann kannst du dafür online Code-Editoren benutzen.
-Probiere "Hallo Welt" auf [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), [Glitch](https://glitch.com/edit/#!/remix/starter-react-template) oder [Stackblitz](https://stackblitz.com/fork/react) aus.
+Wenn du aus Interesse ein wenig mit React herumspielen möchtest, kannst du dafür online Code-Sandkasten benutzen.
```
Comment (ph1p) [inline content/docs/getting-started.md:39]:
```suggestion
Wenn du aus Interesse ein wenig mit React herumspielen möchtest, kannst du dafür den online Code-Sandkasten benutzen.
```

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,47 +18,47 @@ redirect_from:
...
+Wenn du lieber deinen eigenen Texteditor verwenden möchtest, kannst du [diese HTML-Datei](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html) herunterladen, lokal bearbeiten und in deinem Browser aufrufen. Die Datei führt eine langsame Codeumwandlung in der Laufzeit aus, daher würden wir sie nur für einfachere Demonstrationszwecke empfehlen.
 
-### React zu einer Webseite hinzufügen {#add-react-to-a-website}
+### Füge React zu einer Webseite hinzu {#add-react-to-a-website}
 
-Du kannst React innerhalb einer Minute zu einer HTML-Seite hinzufügen. Danach kannst du entweder die Präsenz von React immer mehr erweitern oder es nur für ein paar Bestandteile wie Formulare o. ä. benutzen.
+Du kannst [React in einer Minute zu einer beliebigen HTML-Seite hinzufügen](/docs/add-react-to-a-website.html). Danach kannst du entweder die React-Anteile deiner Applikation stetig erweitern, oder es nur für gewisse in sich geschlossene, dynamische Bestandteile benutzen (zum Beispiel für Formulare).
```
Comment (ph1p) [inline content/docs/getting-started.md:46]:
```suggestion
Du kannst [React in einer Minute zu einer beliebigen HTML-Seite hinzufügen](/docs/add-react-to-a-website.html). Danach kannst du entweder die React-Anteile deiner Anwendung stetig erweitern, oder es nur für gewisse in sich geschlossene, dynamische Bestandteile benutzen (zum Beispiel für Formulare).
```
Maybe we should use "Anwendung". "(App)likation" sounds a little bit denglish.

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,47 +18,47 @@ redirect_from:
...
 
-### Erstelle eine neue React App {#create-a-new-react-app}
+### Erstelle eine neue React-App {#create-a-new-react-app}
 
 Am Besten fängst du ein neues React-Projekt mit [einer einfachen HTML Seite und ein paar Script-Tags an](/docs/add-react-to-a-website.html). Innerhalb einer Minute hast du alles fertig aufgesetzt!
 
-Wenn deine Anwendung wächst, möchtest du vielleicht ein stärker integriertes Setup in Betracht ziehen. Es gibt mehrere JavaScript-Werkzeuge, die wir für größere Anwendungen empfehlen. Jedes von ihnen kommt mit wenig bis garkeiner Vorkonfiguration aus und lässt dich trotzdem den vollen Funktionsumfang des React-Ökosystems genießen. [Erfahre wie.](/docs/create-a-new-react-app.html) 
+Wenn deine Anwendung wächst, wirst du womöglich ein stärker integriertes Setup in Betracht ziehen. Es gibt mehrere JavaScript-Werkzeugsammlungen, die wir für größere Anwendungen empfehlen können. Jede dieser Sammlungen kommt mit minimaler bis gar keiner Vorkonfiguration aus. Trotzdem wird sie dir den vollen Funktionsumfang des React-Ökosystems bieten. [Erfahre mehr darüber.](/docs/create-a-new-react-app.html)
```
Comment (ph1p) [inline content/docs/getting-started.md:52]:
```suggestion
Wenn deine Anwendung wächst, wirst du womöglich ein stärker integriertes Setup in Betracht ziehen. Es gibt mehrere JavaScript-Werkzeugsammlungen, die wir für größere Anwendungen empfehlen können. Jede dieser Sammlungen kommt mit minimaler bis gar keiner Vorkonfiguration aus. Trotzdem werden sie dir den vollen Funktionsumfang des React-Ökosystems bieten. [Erfahre mehr darüber.](/docs/create-a-new-react-app.html) 
```
We are talking about mutiple toolsets, or?

**Review by ph1p — CHANGES_REQUESTED** [review]

**Comment by ph1p** [comment]
Thank you! I have only added three small suggestions.

File: content/docs/getting-started.md (hunk 3 above)
Comment (mlohrer) [reply content/docs/getting-started.md:52]: Grammatically, the previous sentence uses the singular, and you would now have a choice between singular and plural. Either is fine, as long as it's consistent with the general style!

File: content/docs/getting-started.md (hunk 1 above)
Comment (mlohrer) [reply content/docs/getting-started.md:39]: Some Bavarian habits snuck in here.

**Review by ph1p — APPROVED** [review]

**Pull request merged**